File: MyLanguageWeb/src/main/resources/static/js/classes/dto/entity/word_in_collection/request/word_in_collection_add_request_dto.js
export class WordInCollectionAddRequestDTO {
    #id;
    #wordId;
    #customerCollectionId;

    getId() {
        return this.#id;
    }

    setId(id) {
        this.#id = id;
    }

    getWordId() {
        return this.#wordId;
    }

    setWordId(wordId) {
        this.#wordId = wordId;
    }

    getCustomerCollectionId() {
        return this.#customerCollectionId;
    }

    setCustomerCollectionId(customerCollectionId) {
        this.#customerCollectionId = customerCollectionId;
    }
}

File: MyLanguageWeb/src/main/java/ru/herooo/mylanguageweb/dto/entity/wordincollection/request/WordInCollectionAddRequestDTO.java
package ru.herooo.mylanguageweb.dto.entity.wordincollection.request;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import com.fasterxml.jackson.databind.annotation.JsonSerialize;
import ru.herooo.mylanguageweb.json.l.LongDeserializer;
import ru.herooo.mylanguageweb.json.l.LongSerializer;

public class WordInCollectionAddRequestDTO {
    @JsonSerialize(using = LongSerializer.class)
    @JsonDeserialize(using = LongDeserializer.class)
    @JsonProperty("word_id")
    private long wordId;

    @JsonSerialize(using = LongSerializer.class)
    @JsonDeserialize(using = LongDeserializer.class)
    @JsonProperty("customer_collection_id")
    private long customerCollectionId;

    @JsonProperty("auth_key")
    private String authKey;

    public long getWordId() {
        return wordId;
    }

    public void setWordId(long wordId) {
        this.wordId = wordId;
    }

    public long getCustomerCollectionId() {
        return customerCollectionId;
    }

    public void setCustomerCollectionId(long customerCollectionId) {
        this.customerCollectionId = customerCollectionId;
    }

    public String getAuthKey() {
        return authKey;
    }

    public void setAuthKey(String authKey) {
        this.authKey = authKey;
    }
}
